Sources/Compiler/Parse/ParserState.swift

```diff
@@ -54,6 +54,9 @@ struct ParserState {
     self.currentIndex = lexer.source.contents.startIndex
   }
 
+  /// The character stream from the current index.
+  var characterStream: Substring { lexer.source.contents[currentIndex...] }
+
   /// Indicates whether the parser is at global scope.
   var atGlobalScope: Bool { atModuleScope || atNamespaceScope }
```

Comment:
```suggestion
  var isAtGlobalScope: Bool { isAtModuleScope || isAtNamespaceScope }
```

I realize you didn't do anything with this in this PR, but these names don't seem to follow the guidelines.  Also, I think this definition of “global” is a little surprising.  Are static members of types "at global scope?"  if not, why not.

Reply:
That property indicates whether the parser is parsing at the global scope, which does not imply that things parsed in type scope can't end up being considered "global" (see `Program.isGlobal(_:)`).